### PR TITLE
Add workflows

### PR DESCRIPTION
**Pull request by FWuellhorst**
I added the workflows to the package by following AixLib and Filips workflows.

### CODE REVIEW

**Comment by FWuellhorst** [comment]
@dnikolay-ebc I will also add the semantic-release directly here.

**Review by dnikolay-ebc — APPROVED** [review]
Looks good from my side

File: CHANGELOG.md [hunk 1]
```diff
@@ -41,3 +41,6 @@
```
Comment (dnikolay-ebc) [inline CHANGELOG.md:41]: Add line `<!--next-version-placeholder-->` here

File: setup.py [hunk 2]
```diff
@@ -28,7 +28,7 @@
     long_description=long_description,
     long_description_content_type='text/markdown',
     url='https://github.com/RWTH-EBC/AixCaliBuHA',
-    download_url='https://github.com/RWTH-EBC/AixCaliBuHA/archive/refs/tags/0.2.2.tar.gz',
+    download_url='https://github.com/RWTH-EBC/AixCaliBuHA/archive/refs/tags/0.2.3.tar.gz',
```
Comment (dnikolay-ebc) [inline setup.py:31]: can you use `f'https://github.com/RWTH-EBC/AixCaliBuHA/archive/refs/tags/{__version__}.tar.gz'` here?

**Review by dnikolay-ebc — CHANGES_REQUESTED** [review]
You should use conventional commits to benefit from conventional releases.

File: setup.py (hunk 2 above)
Comment (FWuellhorst) [reply setup.py]: Nice! Yes!!

File: CHANGELOG.md (hunk 1 above)
Comment (FWuellhorst) [reply CHANGELOG.md:41]: Like this? :)

**Comment by FWuellhorst** [comment]
> You should use conventional commits to benefit from conventional releases.

Yes, totally agree. I started in the last commit.

**Review by dnikolay-ebc — APPROVED** [review]

**Pull request merged**